src/form/v2/birth/forms/pages/child.ts

```diff
@@ -222,7 +222,7 @@ export const child = defineFormPage({
       validation: [
         {
           message: {
-            defaultMessage: 'Please enter a valid date',
+            defaultMessage: 'Must be a valid Birthdate',
```

Comment:
```suggestion
            defaultMessage: 'Must be a valid birthdate',
```

Reply:
Should birthdate be lowercase? Do we need to update client.csv btw?

Reply:
It is exactly as "Must be a valid Birthdate" in E2E test suit. Hence it is as it is.